chevah_build

```diff
@@ -34,14 +31,39 @@ PYTHON_BUILD_VERSION=2.7.10
 PYSQLITE_VERSION=2.6.3
 SQLITE_VERSION=3.8.1
 
+REDISTRIBUTABLE_VERSION="9.00.30729.6161"
```

Comment:
I had to check out the changes to get an idea what is this about. Perhaps a comment would be handy here?